Assets/MixedRealityToolkit/_Core/Utilities/ProcessResult.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
```

Comment:
Should this be in definitions?  It's a Struct after all.

Reply:
Sure I'll move any structs in the utilities folder into definitions, but structs that are related specifically to a utility feature needs to stay together.